core/amber/src/main/python/core/runnables/main_loop.py

```diff
@@ -160,24 +161,32 @@ def process_input_tuple(self) -> None:
                 self.context.tuple_processing_manager.current_input_port_id
             )
 
-        for output_tuple in self.process_tuple_with_udf():
+        for output_data in self.process_tuple_with_udf():
```

Comment:
process_tuple_with_udf => process_data_with_udf?

Reply:
This function will only process tuple, updated.